2024-06-13 10:52:45,021 INFO    StreamThr :30776 [internal.py:wandb_internal():89] W&B internal server running at pid: 30776, started at: 2024-06-13 10:52:45.020520
2024-06-13 10:52:45,022 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status
2024-06-13 10:52:45,029 INFO    WriterThread:30776 [datastore.py:open_for_write():85] open: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\run-5nqz3oyx.wandb
2024-06-13 10:52:45,030 DEBUG   SenderThread:30776 [sender.py:send():379] send: header
2024-06-13 10:52:45,191 DEBUG   SenderThread:30776 [sender.py:send():379] send: run
2024-06-13 10:52:49,121 INFO    SenderThread:30776 [retry.py:__call__():172] Retry attempt failed:
urllib3.exceptions.SSLError: [SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:1091)

The above exception was the direct cause of the following exception:

urllib3.exceptions.ProxyError: ('Unable to connect to proxy', SSLError(SSLError(1, '[SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:1091)')))

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Reseach Software\envs\fl\lib\site-packages\requests\adapters.py", line 497, in send
    chunked=chunked,
  File "D:\Reseach Software\envs\fl\lib\site-packages\urllib3\connectionpool.py", line 848, in urlopen
    method, url, error=new_e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "D:\Reseach Software\envs\fl\lib\site-packages\urllib3\util\retry.py", line 515, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLError(1, '[SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:1091)'))))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Reseach Software\envs\fl\lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "D:\Reseach Software\envs\fl\lib\site-packages\wandb\sdk\internal\internal_api.py", line 323, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "D:\Reseach Software\envs\fl\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "D:\Reseach Software\envs\fl\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "D:\Reseach Software\envs\fl\lib\site-packages\wandb\sdk\lib\gql_request.py", line 55, in execute
    request = self.session.post(self.url, **post_args)
  File "D:\Reseach Software\envs\fl\lib\site-packages\requests\sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
  File "D:\Reseach Software\envs\fl\lib\site-packages\requests\sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Reseach Software\envs\fl\lib\site-packages\requests\sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "D:\Reseach Software\envs\fl\lib\site-packages\requests\adapters.py", line 513, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLError(1, '[SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:1091)'))))
2024-06-13 10:52:50,245 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: keepalive
2024-06-13 10:52:55,278 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: keepalive
2024-06-13 10:53:00,319 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: keepalive
2024-06-13 10:53:04,441 INFO    SenderThread:30776 [dir_watcher.py:__init__():211] watching files in: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files
2024-06-13 10:53:04,441 INFO    SenderThread:30776 [sender.py:_start_run_threads():1124] run started: 5nqz3oyx with start time 1718247165.021518
2024-06-13 10:53:04,442 DEBUG   SenderThread:30776 [sender.py:send_request():406] send_request: summary_record
2024-06-13 10:53:04,442 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:04,442 INFO    SenderThread:30776 [sender.py:_save_file():1375] saving file wandb-summary.json with policy end
2024-06-13 10:53:04,450 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: check_version
2024-06-13 10:53:04,450 DEBUG   SenderThread:30776 [sender.py:send_request():406] send_request: check_version
2024-06-13 10:53:04,937 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: run_start
2024-06-13 10:53:04,967 DEBUG   HandlerThread:30776 [system_info.py:__init__():31] System info init
2024-06-13 10:53:04,968 DEBUG   HandlerThread:30776 [system_info.py:__init__():46] System info init done
2024-06-13 10:53:04,968 INFO    HandlerThread:30776 [system_monitor.py:start():181] Starting system monitor
2024-06-13 10:53:04,968 INFO    SystemMonitor:30776 [system_monitor.py:_start():145] Starting system asset monitoring threads
2024-06-13 10:53:04,968 INFO    HandlerThread:30776 [system_monitor.py:probe():201] Collecting system info
2024-06-13 10:53:04,974 INFO    SystemMonitor:30776 [interfaces.py:start():190] Started cpu monitoring
2024-06-13 10:53:04,974 INFO    SystemMonitor:30776 [interfaces.py:start():190] Started disk monitoring
2024-06-13 10:53:04,974 INFO    SystemMonitor:30776 [interfaces.py:start():190] Started gpu monitoring
2024-06-13 10:53:04,975 INFO    SystemMonitor:30776 [interfaces.py:start():190] Started memory monitoring
2024-06-13 10:53:04,983 INFO    SystemMonitor:30776 [interfaces.py:start():190] Started network monitoring
2024-06-13 10:53:04,991 DEBUG   HandlerThread:30776 [system_info.py:probe():195] Probing system
2024-06-13 10:53:04,992 DEBUG   HandlerThread:30776 [system_info.py:_probe_git():180] Probing git
2024-06-13 10:53:05,072 DEBUG   HandlerThread:30776 [system_info.py:_probe_git():188] Probing git done
2024-06-13 10:53:05,072 DEBUG   HandlerThread:30776 [system_info.py:probe():240] Probing system done
2024-06-13 10:53:05,072 DEBUG   HandlerThread:30776 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.19041-SP0', 'python': '3.7.12', 'heartbeatAt': '2024-06-13T02:53:04.991696', 'startedAt': '2024-06-13T02:52:45.014540', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'D:\\Research-Projects-jupyter\\PFLlib\\system\\main.py', 'codePath': 'system\\main.py', 'git': {'remote': 'https://github.com/TsingZ0/PFLlib.git', 'commit': '004a405da7c159e843d98b2a53e2114d4f859912'}, 'email': '1731485214@qq.com', 'root': 'D:/Research-Projects-jupyter/PFLlib', 'host': 'DESKTOP-4LOGPS7', 'username': 'paddeyzhang', 'executable': 'D:\\Reseach Software\\envs\\fl\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 2500.0, 'min': 0.0, 'max': 2500.0}, 'cpu_freq_per_core': [{'current': 2500.0, 'min': 0.0, 'max': 2500.0}], 'disk': {'total': 199.99901962280273, 'used': 68.89519882202148}, 'gpu': 'NVIDIA GeForce RTX 3070', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3070', 'memory_total': 8589934592}], 'memory': {'total': 15.842487335205078}}
2024-06-13 10:53:05,072 INFO    HandlerThread:30776 [system_monitor.py:probe():211] Finished collecting system info
2024-06-13 10:53:05,072 INFO    HandlerThread:30776 [system_monitor.py:probe():214] Publishing system info
2024-06-13 10:53:05,072 DEBUG   HandlerThread:30776 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-06-13 10:53:05,073 DEBUG   HandlerThread:30776 [system_info.py:_save_pip():67] Saving pip packages done
2024-06-13 10:53:05,073 DEBUG   HandlerThread:30776 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-06-13 10:53:05,074 ERROR   HandlerThread:30776 [system_info.py:_save_conda():85] Error saving conda packages: [WinError 2] 系统找不到指定的文件。
Traceback (most recent call last):
  File "D:\Reseach Software\envs\fl\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 82, in _save_conda
    ["conda", "env", "export"], stdout=f, stderr=subprocess.DEVNULL
  File "D:\Reseach Software\envs\fl\lib\subprocess.py", line 339, in call
    with Popen(*popenargs, **kwargs) as p:
  File "D:\Reseach Software\envs\fl\lib\subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "D:\Reseach Software\envs\fl\lib\subprocess.py", line 1207, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] 系统找不到指定的文件。
2024-06-13 10:53:05,077 DEBUG   HandlerThread:30776 [system_info.py:_save_conda():86] Saving conda packages done
2024-06-13 10:53:05,078 INFO    HandlerThread:30776 [system_monitor.py:probe():216] Finished publishing system info
2024-06-13 10:53:05,081 DEBUG   SenderThread:30776 [sender.py:send():379] send: files
2024-06-13 10:53:05,081 INFO    SenderThread:30776 [sender.py:_save_file():1375] saving file wandb-metadata.json with policy now
2024-06-13 10:53:05,092 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: stop_status
2024-06-13 10:53:05,092 DEBUG   SenderThread:30776 [sender.py:send_request():406] send_request: stop_status
2024-06-13 10:53:05,444 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_created():271] file/dir created: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\conda-environment.yaml
2024-06-13 10:53:05,445 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_created():271] file/dir created: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\wandb-metadata.json
2024-06-13 10:53:05,445 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_created():271] file/dir created: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\wandb-summary.json
2024-06-13 10:53:05,445 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_created():271] file/dir created: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\requirements.txt
2024-06-13 10:53:05,706 DEBUG   SenderThread:30776 [sender.py:send():379] send: telemetry
2024-06-13 10:53:06,452 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_created():271] file/dir created: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\output.log
2024-06-13 10:53:06,466 INFO    wandb-upload_0:30776 [upload_job.py:push():131] Uploaded file C:\Users\ly\AppData\Local\Temp\tmpb_q8fo17wandb\u4u96h1a-wandb-metadata.json
2024-06-13 10:53:08,480 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_modified():288] file/dir modified: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\output.log
2024-06-13 10:53:09,768 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:11,499 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_modified():288] file/dir modified: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\output.log
2024-06-13 10:53:14,812 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:18,448 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: partial_history
2024-06-13 10:53:18,449 DEBUG   SenderThread:30776 [sender.py:send():379] send: history
2024-06-13 10:53:18,449 DEBUG   SenderThread:30776 [sender.py:send_request():406] send_request: summary_record
2024-06-13 10:53:18,451 INFO    SenderThread:30776 [sender.py:_save_file():1375] saving file wandb-summary.json with policy end
2024-06-13 10:53:18,563 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_modified():288] file/dir modified: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\wandb-summary.json
2024-06-13 10:53:18,626 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: pause
2024-06-13 10:53:18,626 INFO    HandlerThread:30776 [handler.py:handle_request_pause():728] stopping system metrics thread or process
2024-06-13 10:53:18,626 INFO    HandlerThread:30776 [system_monitor.py:finish():190] Stopping system monitor
2024-06-13 10:53:18,626 DEBUG   SystemMonitor:30776 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2024-06-13 10:53:18,627 INFO    HandlerThread:30776 [interfaces.py:finish():202] Joined cpu monitor
2024-06-13 10:53:18,627 DEBUG   SystemMonitor:30776 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2024-06-13 10:53:18,634 INFO    HandlerThread:30776 [interfaces.py:finish():202] Joined disk monitor
2024-06-13 10:53:18,634 DEBUG   SystemMonitor:30776 [system_monitor.py:_start():170] Publishing last batch of metrics
2024-06-13 10:53:18,690 INFO    HandlerThread:30776 [interfaces.py:finish():202] Joined gpu monitor
2024-06-13 10:53:18,690 INFO    HandlerThread:30776 [interfaces.py:finish():202] Joined memory monitor
2024-06-13 10:53:18,690 INFO    HandlerThread:30776 [interfaces.py:finish():202] Joined network monitor
2024-06-13 10:53:18,690 DEBUG   SenderThread:30776 [sender.py:send():379] send: stats
2024-06-13 10:53:20,094 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: stop_status
2024-06-13 10:53:20,097 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:20,382 DEBUG   SenderThread:30776 [sender.py:send_request():406] send_request: stop_status
2024-06-13 10:53:20,580 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_modified():288] file/dir modified: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\config.yaml
2024-06-13 10:53:21,594 INFO    Thread-16 :30776 [dir_watcher.py:_on_file_modified():288] file/dir modified: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240613_105245-5nqz3oyx\files\output.log
2024-06-13 10:53:25,684 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:30,717 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:35,109 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: stop_status
2024-06-13 10:53:35,109 DEBUG   SenderThread:30776 [sender.py:send_request():406] send_request: stop_status
2024-06-13 10:53:36,465 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
2024-06-13 10:53:41,519 DEBUG   HandlerThread:30776 [handler.py:handle_request():144] handle_request: status_report
